helix-rest/src/test/java/org/apache/helix/rest/client/TestCustomRestClient.java

```diff
@@ -158,11 +158,12 @@ public void testPostRequestFormat() throws IOException {
     final String echoServer = "http://httpbin.org/post";
     CustomRestClientImpl customRestClient = new CustomRestClientImpl(HttpClients.createDefault());
     HttpResponse response = customRestClient.post(echoServer, Collections.emptyMap());
-    JsonNode json = customRestClient.getJsonObject(response);
+    if (response.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
```

Comment:
Shouldn't we assert in else block as well? Or we are expecting any other status code is fine?

Reply:
Thanks for review. Test is ECHOing the request to outside server. If you can't reach the outside server, test used to fail. That is why we validate only if we can successfully reach the end-point. Else part, we can't validate.